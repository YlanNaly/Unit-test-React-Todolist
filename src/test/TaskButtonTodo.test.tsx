import React from "react";
import TasksButtonToDo from "../todolist/TaskButtonTodo";
import {fireEvent, render} from "@testing-library/react";

it("render input in snapshot" , ()=>{
    const task = [{ id: 1, text: "Manger!", done: true }]
    const tree = render(<TasksButtonToDo addTasks={null} deleteTask3={null} task={task} toggleTask={null} deleteTask={()=>jest.fn()}/>) ;
    const input = tree.getByTestId("deleteTask");
    expect(input).toMatchSnapshot();
})