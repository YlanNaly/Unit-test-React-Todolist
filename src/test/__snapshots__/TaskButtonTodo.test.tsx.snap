// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render input in snapshot 1`] = `
<span
  class="forme"
  data-testid="deleteTask"
  role="button"
  style="padding: 5px; margin-left: 20px; font-size: 20px;"
>
  <input
    type="checkbox"
  />
</span>
`;
